src/DynamicDriver.cs

```diff
@@ -199,11 +199,13 @@ orderby e.LogicalName
                                                                     var enumValue = x.SanitisedLabel;
                                                                     if (string.IsNullOrEmpty(x.SanitisedLabel))
                                                                     {
-                                                                        enumValue = $"_{x.Value}";
+																		//When the value is a negative number, replace '-' with '_'.
```

Comment:
These blanks are automatically generated because this file uses a mix of `space` and `tab` for indentation.

Reply:
@JadynWong - Thank you for the PR. I will look into this next week.